### PR TITLE
Feature/devsu 1431 add msi data to summary

### PR DESCRIPTION
**Pull request by Anna-log7**
This is to handle the MSI data in the genomic summary that will be loaded into IPR soon. I also added full typing to the genomic summary

### CODE REVIEW

File: app/views/ReportView/components/GenomicSummary/index.tsx [hunk 1]
```diff
@@ -13,21 +12,26 @@ import { useSnackbar } from 'notistack';
...
+import {
+  MsiType,
+  PatientInformationType,
+  GeneVariantType,
+  TumourSummaryType,
+  MicrobialType,
+} from './types.d';
+import { MutationBurdenType, ComparatorType } from '../MutationBurden/types.d';
+import MutationSignatureType from '../MutationSignatures/types.d';
+import { ImmuneType } from '../Immune/types';
```
Comment (kttkjl) [inline app/views/ReportView/components/GenomicSummary/index.tsx:34]: weird how this works, but `.d` is required of all the other types ...

File: app/components/Image/index.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React, { useState, useCallback } from 'react';
 import { Typography, Fade } from '@material-ui/core';
 
-import ImageType from './types';
+import ImageType from './types.d';
```
[the remark this answers is not in the document]
Comment (Anna-log7) [reply app/components/Image/index.tsx:4]: oh that probably would. It was working on/off so I never thought about it really

**Review by kttkjl — COMMENTED** [review]
- there are inconsistencies with imports on `.d` files, some of them work, some of them don't

- MSI addition works

**Review by kttkjl — CHANGES_REQUESTED** [review]
there are inconsistencies with imports on .d files, some of them work without the .d in the path, some of them don't

MSI addition works

Github reviews are still weird to ...

File: app/views/ReportView/components/GenomicSummary/index.tsx (hunk 1 above)
Comment (Anna-log7) [reply app/views/ReportView/components/GenomicSummary/index.tsx:34]: I've removed the `.d` extension now

**Review by kttkjl — APPROVED** [review]

**Review by kttkjl — APPROVED** [review]

**Pull request merged**